shadow-cljs.edn

```diff
@@ -77,6 +77,7 @@
      #shadow/env "STATUS_BACKEND_SERVER_IMAGE_SERVER_URI_PREFIX"
      status-im.config/STATUS_BACKEND_SERVER_ROOT_DATA_DIR #shadow/env
                                                            "STATUS_BACKEND_SERVER_ROOT_DATA_DIR"
+     status-im.config/SENTRY_DSN_STATUS_GO #shadow/env "SENTRY_DSN_STATUS_GO"
```

Comment:
this should be in the release block below and not here :)

Reply:
Thanks for finding that @siddarthkay! Shouldn't the var be in the shadow-cljs dev environment as well? Similar to how we do with most other vars.

Reply:
it probably should not be.
If a case should arise that we need `SENTRY_DSN_STATUS_GO` mapping in `jsbundle` for debug builds then yes, else no.
I think previously we just added the mapping to both `debug` and `release` schemes but It did not matter because when debug builds were created those vars would be empty :)

Reply:
Cool! I'll then just leave a comment in `status-im.config` besides the Clojure var to help a dev figure out that to test Sentry during development they need to change the shadow-cljs `:mobile` `:dev` target.

Reply:
Done